src/musicPlayer.java

```diff
@@ -0,0 +1,37 @@
+import components.queue.Queue;
+import components.queue.Queue1L;
+
+/** */
+public class musicPlayer {
+    /** */
+
+    private Queue<String> playlist;
+
+    /** */
+    public musicPlayer() {
+        this.playlist = new Queue1L<>();
+    }
+
+    /** */
+    public void queue(String song) {
+        this.playlist.enqueue(song);
+    }
+
+    public void remove(String song) {
+        Queue<String> temp = new Queue1L<>();
+
+        while (this.playlist.length() > 0) {
+            String currentSong = this.playlist.dequeue();
+
+            if (!currentSong.equals(song)) {
+                temp.enqueue(currentSong);
+            }
+            this.playlist = temp;
+        }
+    }
+
+    /** */
+    public static void main(String[] args) {
+        String song = null;
```

Comment:
If you want to show you component in action, you need something like the following:
```java
MusicPlaylist p = new MusicPlaylist();
p.queue("example");
```